src/store/BarrierStore.js

```diff
@@ -1,11 +1,13 @@
-import { observable, action, computed } from 'mobx';
+import { observable, action, computed, when } from 'mobx';
 import PriceLineStore from './PriceLineStore';
 import ShadeStore from './ShadeStore';
 import PendingPromise from '../utils/PendingPromise';
 import PriceLine from '../components/PriceLine.jsx';
 import Shade from '../components/Shade.jsx';
 import { isValidProp } from '../utils';
 
+const LINE_OFFSET_HEIGHT = 30;
```

Comment:
Why is it a global variable?

Reply:
@morteza-binary the shade jump size pixel is fixed. and it's around `30px`

Reply:
@ I see, but it can be a constant property of the class instead of being global unless it is used by other classes too.

Reply:
@morteza-binary aha, I get your point